[SC-10229] update github-discovery docs (#836)
RoadieHQ/marketing-site: content/docs/getting-started/install-github-app/index.md

5. Choose your GitHub organization and follow the steps to install the app.

ℹ️ &nbsp;If you are not an owner of your GitHub organization, you will need to ask an owner to approve the App installation and then contact us to enable the App.

> Roadie will ingest any `catalog-info.yaml` files added to your Github repos. If you'd like to have more granular control (e.g. you don't want to call them catalog-info.yaml, or you'd like to have them inside a folder etc.) over what files Roadie should ingest, go to the [GitHub discovery docs](/docs/integrations/github-discovery/).

Reviewer: Here is a suggested rewording of this paragraph.

If you already have catalog-info.yaml files in your repositories, Roadie will attempt to ingest them into the Backstage catalog. If you need to have more granular control over the location of the catalog files, you can do that by following the steps here [GitHub discovery docs](/docs/integrations/github-discovery/)

Author: > If you already have catalog-info.yaml files in your repositories, Roadie will attempt to ingest them into the Backstage catalog. 

For this they need to use the scraping to ingest existing catalog-info files

> Roadie will listen to your github and add any `catalog-info.yaml` files added to your repos by default. If you'd like to have more granular control(e.g. you don't want to call them catalog-info.yaml, or you'd like to have them inside a folder etc.) over what files should roadie add to the catalog go to the [github discovery docs](/docs/integrations/github-discovery/)

Reviewer 2: ```suggestion
Roadie will ingest any `catalog-info.yaml` files added to your Github repos. If you'd like to have more granular control(e.g. you don't want to call them catalog-info.yaml, or you'd like to have them inside a folder etc.) over what files should roadie add to the catalog go to the [github discovery docs](/docs/integrations/github-discovery/)
```
